app/controllers/completed_tasks_controller.rb

```diff
@@ -5,11 +5,29 @@ def new
   end
 
   def create
-    if (params[:is_checked].to_s == "true")
-      current_user.completed_tasks.build(completed_task_params)
+    current_user.completed_tasks.build(completed_task_params)
+    if current_user.save
+      respond_to do |format|
+        format.html do
+          redirect_to root_url, notice: "Save process completed!"
+        end
+        format.json do
+          render json: nil, status: :ok
+        end
+      end
     else
-      current_user.completed_tasks.where(completed_task_params).destroy_all
+      respond_to do |format|
+        format.html do
+          redirect_to dashboard_url, notice: "Save failed"
+        end
+        format.json do
+          render json: { success: false } 
+        end
+      end
     end
+  end
+  def destroy
```

Comment:
Assignment Branch Condition size for destroy is too high. [19.03/15]<br>Use empty lines between defs.